attributes/default.rb

```diff
@@ -20,3 +20,4 @@
 default['build-essential']['compile_time'] = false
 default['build-essential']['mingw32']['path'] = "#{ENV['SYSTEMDRIVE']}\\mingw32"
 default['build-essential']['mingw64']['path'] = "#{ENV['SYSTEMDRIVE']}\\mingw64"
+default['build-essential']['msys64']['path'] = "#{ENV['SYSTEMDRIVE']}\\msys64"
```

Comment:
Since it appears the same MSYS2 is used on 32-bit and 64-bit installations wouldn't it be less confusing to refer to everything as `msys2` or `msys`?

```ruby
default['build-essential']['msys2']['path'] = "#{ENV['SYSTEMDRIVE']}\\msys2"
```